commands/create_command.py

```diff
@@ -39,4 +47,18 @@ def __create_tables(self):
 
         for table in self.data[SchemaKeys.TABLES]:
             t_path = os.path.join(path, table[SchemaKeys.NAME])
-            os.makedirs(t_path, exist_ok = True)
+            os.makedirs(t_path, exist_ok=True)
+
+    def create_table_schema(self, t_path, table):
+        file: TextIO = open(os.path.join(t_path, table[SchemaKeys.NAME] + " schema.json"), 'w')
+        json.dump(table, file)
+        file.close()
+
+    def create_indices(self, t_path, table):
```

Comment:
What are you going to do in Set or Get ? You will load all indices in the memory which is overhead. 

I recommend every index in folder. Every Key in file with the name of the keys